services/persistence/main.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+import os
+import manifesto_data
+
+from flask import Flask, jsonify, request
+import sqlite3
+import urllib.parse
+
+app = Flask(__name__)
+
+DEBUG = True  # os.environ.get('DEBUG') is not None
+VERSION = 0.1
+
+
+@app.route("/", methods=['POST'])
+def index():
+    return jsonify({})
+
+
+@app.route("/texts", methods=['GET'])
+def texts():
+    n_texts = urllib.parse.quote(request.args.get('n'))
+    text_data = {'data': get_texts(n_texts)}
+    return jsonify(text_data)
+
+
+def get_texts(n_texts):
+    conn = sqlite3.connect('/tmp/db/test.db')
+    c = conn.cursor()
+    return [
+        {
+            'statement': statement,
+            'label': label,
+            'source': source
+        } for statement, label, source, _, _ in c.execute(
+        """
+        SELECT
+            t.statement,
+            l.label,
+            l.source,
+            t.created_at,
+            l.created_at
+        FROM texts t
+        INNER JOIN labels l ON l.texts_id = t.id
+        LIMIT ?""",
+        (n_texts, )
+        )
+    ]
+
+
+def create_manifesto_storage(texts, labels):
+    conn = sqlite3.connect('/tmp/db/test.db')
+    c = conn.cursor()
+
+    c.execute('DROP TABLE IF EXISTS texts')
+    c.execute(
+        '''CREATE TABLE texts (
+                id INTEGER PRIMARY KEY AUTOINCREMENT,
+                created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+                statement TEXT NOT NULL
+            )
+        '''
+    )
+    c.execute('DROP TABLE IF EXISTS labels')
+    c.execute(
+        '''CREATE TABLE labels (
+                id INTEGER PRIMARY KEY AUTOINCREMENT,
+                texts_id INTEGER NOT NULL,
+                label INTEGER NOT NULL,
+                source text NOT NULL,
+                created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+                FOREIGN KEY(texts_id) REFERENCES texts(id)
+            )
+        '''
+    )
+
+    pks = []
+    for text in texts:
```

Comment:
das, also erst texte, dann den rest inserten, muss gemacht werden wegen des automatischen key assignments?

Reply:
Genau, müssen erst alle in die primäre Tabelle inserieren um dann die Fremdschlüssel in der Sekundärtabelle füllen zu können.